diff.txt

```diff
@@ -0,0 +1,759 @@
+
+
+blade.portlet.actioncommand-1.0.0.jar
+*******************************************************
+
+
+blade.portlet.blueprint-1.0.0.jar
+*******************************************************
+
+
+blade.portlet.controlpanel-1.0.0.jar
+*******************************************************
+
+
+blade.portlet.ds-1.0.0.jar
+*******************************************************
+
+
+blade.portlet.filter-1.0.0.jar
+*******************************************************
+
+
+blade.portlet.jsp-1.0.0.jar
+*******************************************************
+CHANGED    MANIFEST   <manifest>
+ ADDED      HEADER     Require-Capability:osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://java.sun.com/portlet_2_0))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/aui))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/portlet))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/theme))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/ui))",osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.7))"
+ REMOVED    HEADER     Require-Capability:osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://java.sun.com/portlet_2_0))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/aui))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/portlet))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/theme))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/ui))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/aui))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/portlet))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/theme))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/ui))",osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.7))"
+
+
+blade.portlet.osgiapi-1.0.0.jar
+*******************************************************
+
+
+blade.resourcebundle-1.0.0.jar
+*******************************************************
+CHANGED    MANIFEST   <manifest>
+ REMOVED    HEADER     Provide-Capability:osgi.service;objectClass:List<String>="java.util.ResourceBundle"
+ ADDED      HEADER     Provide-Capability:osgi.service;objectClass:List<String>="java.util.ResourceBundle",liferay.resource.bundle;bundle.symbolic.name="blade.resourcebundle";resource.bundle.base.name="content.Language"
```

Comment:
Not sure why these are different, both should be coming from ResourceBundlePlugin